packages/components/src/utils/config-values.js

```diff
@@ -5,14 +5,16 @@ import { space } from './space';
 import { COLORS } from './colors-values';
 
 const CONTROL_HEIGHT = '36px';
-const CONTROL_PADDING_X = '12px';
 
 const CONTROL_PROPS = {
 	controlSurfaceColor: COLORS.white,
 	controlTextActiveColor: COLORS.theme.accent,
-	controlPaddingX: CONTROL_PADDING_X,
-	controlPaddingXLarge: `calc(${ CONTROL_PADDING_X } * 1.3334)`,
-	controlPaddingXSmall: `calc(${ CONTROL_PADDING_X } / 1.3334)`,
+
+	// These values should be shared with TextControl.
+	controlPaddingX: 12,
+	controlPaddingXSmall: 8,
+	controlPaddingXLarge: 12 * 1.3334, // TODO: Deprecate
```

Comment:
If `ItemGroup` is the only consumer of this variable, I guess it makes sense to just delete it from the common config and inline the value in `ItemGroup`? Also, should we consider aligning `ItemGroup` to the rest of the components and have a `12px` padding too?

Reply:
ItemGroup already does have a 12px padding in the default size, interestingly. We'll probably deprecate the `controlPaddingXLarge` variable and the `large` size of ItemGroup at the same time.